keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,464 @@
+### Removing a KEP from the Release
+
+Throughout the release cycle, a KEP may be removed from the release. This often occurs after `enhancement freeze`, when an at-risk KEP does not meet the specified stage requirements, or after `code freeze`, if all required implementation is not complete. Once these dates occur, the enhancements team will move the associated receipt to the `removed` directory of the release and open a PR to update the repository.
+
+At this point, the KEP is no longer considered part of the release, and an exception must be granted in order for the KEP to be accepted back into the release.
+
+We will enhance `kepctl` to add a `remove` command that will handle the movement of the `receipt`:
+
+```
+$ tree releases
+releases
+└── v1.22
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    |   └── 0002.yaml
+    ├── exception
+    ├── OWNERS
+    ├── proposed
+    |   └── 0004.yaml
+    └── removed
+        └── 0003.yaml
+
+$ kepctl remove 0004 --release v1.22
+==> KEP 0004 moved to removed
+
+$tree releases
+releases
+└── v1.22
+    ├── accepted
+    │   └── 0001.yaml
+    ├── at-risk
+    |   └── 0002.yaml
+    ├── exception
+    ├── OWNERS
+    ├── proposed
+    └── removed
+        └── 0003.yaml
+        └── 0004.yaml
+```
+
+
+### Requesting an Exception for a KEP
+
+To request an exception for the release, the responsible SIG will create a new PR that moves the receipt into the `exception` directory. A PR template  in the `kubernetes/enhancement` repo will capture the relevant information for the exception request. The release team will approve the exception reqeust, but will require confirmation from any impacted reviewers on the PR before it is merged.
+
+
+```
+$ kepctl exception request --release v1.22 --sig api-machinery 1240
+==> api-machinery/1240 moved to exception directory for v1.22.
+==> Please open a pull request to complete this request: link_with_template.
+
+```
+
+The exception request pull request template will contain the questionnaire currently used in the exception request process:
+
+```
+Enhancement name:
+Enhancement status (alpha/beta/stable):
+SIG:
+k/enhancements repo issue #:
+PR #’s:
+Additional time needed (in days):
+Reason this enhancement is critical for this milestone:
+Risks from adding code late: (to k8s stability, testing, etc.)
+Risks from cutting enhancement: (partial implementation, critical customer usecase, etc.)
+```
+
+If an exception request is not granted, the receipt is moved back to the `removed` directory. If the exception is granted, it is moved to the `at-risk` directory until the conditions that caused it to be removed are corrected. `kepctl` will provide convenience methods for this as well:
+
+```
+$ kepctl exception accept --release v1.22 --sig api-machinery 1240
+==> api-machinery/1240 moved to at-risk.
+
+
+$ kepctl exception deny --release v1.22 --sig api-machinery 1240
+==> api-machinery/1240 moved to removed
+```
+
+
+### Generating the release manifest
+
```

Comment:
An issue that I see here is that there is no real-time one-stop artifact anymore. Per the below comment the manifest is manually generated, which means that it will frequently be stale, which seems like a downside as the spreadsheet was always the real-time status that anyone could refer to at any time.

If this is a yaml (not sure), we'll lose clickable links and easy color coding and stats.